src/loader.c

```diff
@@ -1954,8 +2059,7 @@ static void ldr_show_pw_line(struct db_main *db, char *line)
 	if (found && show) {
 		if (source[0])
 			printf("%c%s", db->options->field_sep_char, source);
-		else
-			putchar('\n');
+		putchar('\n');
 	}
```

Comment:
This change still bugs me: Not that I mind it, I just want to understand just WHAT changed that made this change necessary. And I simply can't see what it is! Actually, I'm more confused with the OLD version. Where the heck did it get a newline from?

I tried putting fgets() back (and just running short hashes) and sure as hell, the above change need to be reverted or there will be double linefeeds. But WHY?!